pyproject.toml

```diff
@@ -1,8 +1,41 @@
 [build-system]
-# AVOID CHANGING REQUIRES: IT WILL BE UPDATED BY PYSCAFFOLD!
-requires = ["setuptools>=46.1.0", "setuptools_scm[toml]>=5", "wheel"]
+requires      = ["setuptools>=61.0.0", "wheel"]
 build-backend = "setuptools.build_meta"
 
-[tool.setuptools_scm]
-# See configuration details in https://github.com/pypa/setuptools_scm
-version_scheme = "no-guess-dev"
+[project]
+name = "actinia-api"
+version = "3.3.1"
+description = "API description for actinia-core and official plugins."
+readme = "README.md"
+authors = [
+    { name = "Anika Weinmann"},
+    { name = "Carmen Tawalika"},
+    { name = "Julia Haas"},
+    { name = "Markus Metz"},
+    { name = "Markus Neteler"},
+    { name = "Sören Gebbert"}
+]
+classifiers = [
+    "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+    "Programming Language :: Python :: 3",
+]
+requires-python = ">=3.8"
+keywords = ["processing", "earth observation", "grass gis"]
+dependencies = [
+    "flask-restful-swagger-2"
+]
+
+[project.optional-dependencies]
+test = [
+    "setuptools",
+    "pytest",
+    "ytest-cov"
+]
+
+[project.urls]
+Homepage = "https://github.com/mundialis/actinia-api"
+Tutorial = "https://mundialis.github.io/actinia_core"
+API_Docs = "https://redocly.github.io/redoc/?url=https://actinia.mundialis.de/latest/swagger.json"
+
+[tool.flake8]
+max-line-length = 88
```

Comment:
Why is it set to 88 and not as normal to 80?

Reply:
Because it was the default before in setup.cfg. If I set it to less, there are lint errors. So I would suggest to change this in a separate PR if wanted.

Reply:
okay